### PR TITLE
refine profile readme file

### PR DESCRIPTION
**Pull request by chai2010**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [X] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [X] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
+Unified Programmable Configuration Tech Stack. Home page https://kusionstack.io ([快速开始](https://kusionstack.io/docs/user_docs/getting-started/kubernetes)).
 
-## Official website
+## Repositories description
 
-The main KusionStack Containers site is https://kusionstack.io.
+### Community
 
-## Getting started
+- [community](https://github.com/KusionStack/community) is KusionStack user and developer community 👋
```
Comment (Peefy) [inline profile/README.md]: s/community/Community?

**Review by amyXia1994 — DISMISSED** [review]
lgtm

File: profile/README.md (hunk 1 above)
Comment (chai2010) [reply profile/README.md]: fixed

**Review by amyXia1994 — APPROVED** [review]

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**